src/resources/parser/glb-parser.js

```diff
@@ -1774,6 +1774,7 @@ const createLight = function (gltfLight, node) {
         // - Omni and spot lights use luminous intensity in candela (lm/sr)
         // - Directional lights use illuminance in lux (lm/m2).
         // Current implementation: clapms specified intensity to 0..2 range
+        luminance: gltfLight.intensity,
```

Comment:
see the line below - that's how it should be set, making sure the property exists first.